.circleci/config.yml

```diff
@@ -417,6 +417,14 @@ workflows:
             - test_211_jdk11_jvm
             - test_211_jdk8_js
             - test_211_jdk11_js
+            - test_212_jdk8_jvm
+            - test_212_jdk11_jvm
+            - test_212_jdk8_js
+            - test_212_jdk11_js
+            - test_213_jdk8_jvm
```

Comment:
Similarly:

- I would prefer `test_213` to be moved to the `require` section of `release_213` please

Reply:
here https://github.com/zio/zio/blob/master/.circleci/config.yml#L437